src/containers/Accounts/AMM/AMMAccounts/AMMAccountHeader/AMMAccountHeader.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
```

Comment:
This is in a nested folder `Accounts/AMM/AMMAccounts` - is there anything else in the `Accounts/AMM` folder? If not, then this could be moved to `Accounts/AMMAccounts`

Reply:
yea the folder has other contents.